patches/components-policy-tools-generate_policy_source.py.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/components/policy/tools/generate_policy_source.py b/components/policy/tools/generate_policy_source.py
+index 28db4dd238ad33e4a250d93c1ef90678e5c31ef8..75ce4477f67a9c18de4202913b3b794c769fc800 100755
+--- a/components/policy/tools/generate_policy_source.py
++++ b/components/policy/tools/generate_policy_source.py
+@@ -22,9 +22,10 @@ import sys
+ import textwrap
+ import types
+ from xml.sax.saxutils import escape as xml_escape
++from policy_source_helper import AddBravePolicies
```

Comment:
is it possible to move this below `CHROMIUM_POLICY_KEY` and then import that as well to override it? I think it should work, but not 100% sure

Reply:
Oh, it works! 👍